WireDomain/Sources/WireDomain/Repositories/Conversations/ConversationModelMappings.swift

```diff
@@ -121,7 +121,7 @@ extension WireAPI.Conversation.Member {
             qualifiedTarget: qualifiedTarget?.toDomainModel(),
             target: target,
             conversationRole: conversationRole,
-            service: (service != nil) ? (service!.id, service!.provider) : nil,
+            service: service.map { .init(id: $0.id, provider: $0.provider) },
```

Comment:
In order to make the model equatable, I needed to use a struct instead of a tuple.